### PR TITLE
Make-NFT-Ownable

### LINKED ISSUE #457: Extend NFT to make it non-transferable by holder, only by owner

### PR DESCRIPTION
**Pull request by leomassazza**
Fix #457

### CODE REVIEW

File: packages/core-contracts/contracts/token/ERC721Owned.sol [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+    function safeTransferFrom(
+        address from,
+        address to,
+        uint256 tokenId,
+        bytes memory data
+    ) public virtual override onlyOwner {
+        super.safeTransferFrom(from, to, tokenId, data);
+    }
+
+    function _isApprovedOrOwner(address, uint256) internal view virtual override onlyOwner returns (bool) {
```
Comment (ajsantander) [inline packages/core-contracts/contracts/token/ERC721Owned.sol:33]: Hmmm, not sure about this one. It's weird to have a view function with an access modifier. I'd prefer if it had no modifier and returned `address == owner`.

**Review by ajsantander — APPROVED** [review]
Awesome 😎 

Just a little comment on code style on one of the utility view functions.

File: packages/core-contracts/contracts/token/ERC721Owned.sol (hunk 1 above)
Comment (i-stam) [reply packages/core-contracts/contracts/token/ERC721Owned.sol:33]: agreed, it can be used inside of a modifier instead fi we want this check

**Review by i-stam — APPROVED** [review]

File: packages/core-contracts/contracts/token/ERC721Owned.sol (hunk 1 above)
Comment (leomassazza) [reply packages/core-contracts/contracts/token/ERC721Owned.sol]: Comment addressed. Please confirm this is better (@ajsantander ). Thanks

**Pull request merged**